gui/src/installer/view.rs

```diff
@@ -82,7 +82,7 @@ pub fn define_descriptor<'a>(
                     form::Form::new("Xpub", user_xpub, |msg| {
                         Message::DefineDescriptor(message::DefineDescriptor::UserXpubEdited(msg))
                     })
-                    .warning("Please enter correct xpub")
+                    .warning("Please enter correct xpub for the chosen network")
```

Comment:
That's not a very helpful error message. I think we should go further and say something like:
```suggestion
                    .warning("Please enter an extended key ('xpub') encoded for the configured network (that is, a 'tpub' for test networks)")
```

Also, what about when they import it from their signing device? Can they in all cases get a `tpub`? If so we should probably tell them to do so. I'm not sure they really can on the Ledger, can they?

Reply:
The message becomes too long, I will see if I can change it according to the network choosen previously

Reply:
Since we only ever support mainnet and test network we can be more concise. Something like:
> Please enter a valid `xpub` (or `tpub` for test networks)